src/ElectricityMarketData.jl

```diff
@@ -1,5 +1,64 @@
 module ElectricityMarketData
```

Comment:
keeps using and include in the main file? move this code to another one?

Reply:
will do